lib/src/arg_parser.dart

```diff
@@ -280,9 +282,16 @@ class ArgParser {
       }
     }
 
+    // Make sure the option is not mandatory with a default value
+    if(mandatory && defaultsTo != null) {
+      throw ArgumentError(
+        'The option $name canno\'t be mandatory and have a default value.');
```

Comment:
Typo, should either be `can't` or `cannot`

Reply:
```suggestion
        'The option $name cannot be mandatory and have a default value.');
```